lib/logwriter.c

```diff
@@ -547,6 +545,8 @@ log_writer_stop_watches(LogWriter *self)
 
       self->watches_running = FALSE;
     }
+
+  log_writer_stop_suspend_timer(self);
```

Comment:
Before in the `log_writer_io_handler` did not called `log_writer_stop_suspend_timer` unconditional. Just leave the note, as every other missing place, you first introduced the `log_writer_stop_suspend_timer`.

But I think it is correct to stop the `suspend_timer` there, and even required.